docs/prerequisites.md

```diff
@@ -54,6 +55,30 @@ spec:
     enable: true
 ```
 
+#### With EKS Support
+
+If you want to use the the EKS support in the provider then you will need to enable these features via the configuration file. For example:
+
+```yaml
+apiVersion: bootstrap.aws.infrastructure.cluster.x-k8s.io/v1alpha1
+kind: AWSIAMConfiguration
+spec:
+  bootstrapUser:
+    enable: true
```

Comment:
```suggestion
```

Don't necessarily need the bastion for EKS do you?

Reply:
Good point, thats not required. I have removed it.